src/components/ImportRecordsFlow/stepFour/index.js

```diff
@@ -1,41 +1,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Column from '../../Column';
-import getPreviewDataToImport from '../helpers/getPreviewDataToImport';
```

Comment:
I think this is not used in any other place, if so then remove and clean all related to getPreviewDataToImport